openapi/data_repository_service.swagger.yaml

```diff
@@ -396,3 +405,33 @@ definitions:
       license:
         type: object
         description: License information for the exposed API
+  BundleObject:
+    type: object
+    properties:
+      name:
+        type: string
+        description: >-
+          This represents the Bundle author declared name that must be
+          used when materialising the associated data object and will 
+          override any name directly associated with the object itself.
+          This string MUST NOT contain any slashes.
+      id:
+        type: string
+        description: >-
+          This represents the DRS identifier URI that may point to a Data
```

Comment:
Is this a URI? I expected this to be an `id` string that can be passed to `GET /objects/{object_id}` or `GET /bundles/{bundle_id}`.

Reply:
BundleObject updated with a `drs_uri` that represents a list of possible URIs to obtain the data object or bundle.